shared/src/commonMain/kotlin/co/nimblehq/kmm/template/di/Koin.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.kmm.template.di
+
+import co.nimblehq.kmm.template.di.module.*
+import org.koin.core.KoinApplication
+import org.koin.core.context.startKoin
+import org.koin.dsl.KoinAppDeclaration
+
+fun initKoin(
+    appDeclaration: KoinAppDeclaration = {}
+): KoinApplication {
+    val dataModules = listOf(remoteModule)
+    val domainModules = listOf(repositoryModule, useCaseModule)
```

Comment:
repository should be in data 🙏 

```suggestion
    val dataModules = listOf(remoteModule, repositoryModule)
    val domainModules = listOf(useCaseModule)
```